bench_router_test.go

```diff
@@ -167,18 +167,8 @@ var testRoutes = []string{
 	"/progs/update.bash",
 }
 
-func BenchmarkSliceRouter(b *testing.B) {
-	router := SliceRouter{}
-	benchRouter(&router, b)
-}
-
-func BenchmarkMapRouter(b *testing.B) {
-	router := MapRouter{}
-	benchRouter(&router, b)
-}
-
-func BenchmarkPrefixTreeRouter(b *testing.B) {
-	router := PrefixTreeRouter{}
+func BenchmarkTreeRouter(b *testing.B) {
+	router := TreeRouter{}
```

Comment:
let's call it just "Router" to our router now that we don't have any other type, what to you think?